gradle.properties

```diff
@@ -29,6 +29,7 @@ springVersion=4.2.3.RELEASE
 springWebflowClientVersion=1.0.0
 springWebflowVersion=2.4.2.RELEASE
 springDataVersion=1.8.0.RELEASE
+springSecurityWebVersion=4.0.3.RELEASE
```

Comment:
I think I had to add this to get the ldap stuff running. I'll double verify and make sure to pull this out if possible.